### PR TITLE
Choco added dummy command for provisioner to avoid initial delay.

### PR DESCRIPTION
**Pull request by Darleev**
# Description
Improvement
Added dummy command to cleanup orphaned packages before images fina deployment. 

choco upgrade chocolatey it's just dummy command to make the first start choco on newly created images. We have tested that the solution works in Preflight provisioner deployment.

#### Related issue:
https://github.com/actions/virtual-environments-internal/issues/1120

## Check list
- [+] Related issue / work item is attached
- [-] Tests are written (if applicable)
- [-] Documentation is updated (if applicable)
- [+] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/post-generation/VSConfiguration.ps1 [hunk 1]
```diff
@@ -1,4 +1,7 @@
 $vsInstallRoot = Get-VisualStudioPath
 $devEnvPath = "$vsInstallRoot\Common7\IDE\devenv.exe"
 
-cmd.exe /c "`"$devEnvPath`" /updateconfiguration"
\ No newline at end of file
+cmd.exe /c "`"$devEnvPath`" /updateconfiguration"
+
+# Step to avoid initial delay for choco scripts
+choco upgrade chocolatey
```
Comment (maxim-lobanov) [inline images/win/post-generation/VSConfiguration.ps1:7]: Let's create the separate script for this command. It is not related to Visual Studio actually.

Comment (maxim-lobanov) [inline images/win/post-generation/VSConfiguration.ps1:7]: How was it tested?

Could you please extend explanation comment? For now, it is not clear why `choco upgrade` helps to cleanup orphaned packages

Comment (Darleev) [reply images/win/post-generation/VSConfiguration.ps1:7]: The description has been added, thank you.

Comment (Darleev) [reply images/win/post-generation/VSConfiguration.ps1:7]: Done. thank you

**Review by maxim-lobanov — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Review by AlenaSviridenko — APPROVED** [review]

**Review by vsafonkin — APPROVED** [review]

**Pull request merged**